### ISSUE TITLE
Latest release of 'controls' incorrectly bumps base dep

### ISSUE THREAD

**Issue by vidartf** [jupyter-widgets/ipywidgets#2491]
I think that the latest patch release for the `controls` package might have incorrectly bumped the dependency on `base`, c.f. https://github.com/jupyter-widgets/ipywidgets/commit/2c90130be14f0279deb8d43d79b7bce12f28327c#diff-1e1cb1264372c8fecbcf22d8d8f3af30L37. This is likely an error from lerna automatically bumping that dependency?

The change causes duplication of the `base` package in certain installs with older versions of lab:
```shell
jlpm why @jupyter-widgets/base
=> Found "@jupyter-widgets/base@1.2.5"
info Has been hoisted to "@jupyter-widgets/base"
info Reasons this module exists
   - Hoisted from "jupyterlab-datawidgets#@jupyter-widgets#base"
   - Hoisted from "@jupyter-widgets#jupyterlab-manager#@jupyter-widgets#base"
   - Hoisted from "jupyter-threejs#@jupyter-widgets#base"
   - Hoisted from "jupyter-threejs#jupyter-dataserializers#@jupyter-widgets#base"
   - Hoisted from "jupyterlab-datawidgets#jupyter-datawidgets#@jupyter-widgets#base"
   - Hoisted from "@jupyter-widgets#jupyterlab-manager#@jupyter-widgets#output#@jupyter-widgets#base"
info Disk size without dependencies: "6.94MB"
info Disk size with unique dependencies: "23.02MB"
info Disk size with transitive dependencies: "31.48MB"
info Number of shared dependencies: 29
=> Found "@jupyter-widgets/controls#@jupyter-widgets/base@2.0.1"
info This module exists because "@jupyter-widgets#jupyterlab-manager#@jupyter-widgets#controls" depends on it.
```

**Comment by SylvainCorlay** [comment]
I hate lerna.

**Comment by SylvainCorlay** [comment]
@vidartf if you feel like making a patch release of controls fixing this, please go ahead!

**Comment by vidartf** [comment]
@SylvainCorlay I don't have any rights on the npm registry.

**Comment by SylvainCorlay** [comment]
What is your username?

**Comment by vidartf** [comment]
Same as here

**Comment by SylvainCorlay** [comment]
Args, npmjs does not seem to let me add maintainers at the moment, with some cryptic server error.

**Comment by SylvainCorlay** [comment]
Now it seems ok. Did you get an invite?

**Comment by jasongrout** [comment]
I also got an error. What did you do to get it done?

**Comment by SylvainCorlay** [comment]
You cannot add a member to a single package, but you can manage teams.

**Comment by jasongrout** [comment]
> I think that the latest patch release for the `controls` package might have incorrectly bumped the dependency on `base`, c.f. [2c90130#diff-1e1cb1264372c8fecbcf22d8d8f3af30L37](https://github.com/jupyter-widgets/ipywidgets/commit/2c90130be14f0279deb8d43d79b7bce12f28327c#diff-1e1cb1264372c8fecbcf22d8d8f3af30L37). This is likely an error from lerna automatically bumping that dependency?

I think you are correct on both things.

**Comment by vidartf** [comment]
Patch release of controls out. If you want me to release a patch for the python packages as well (I guess not strictly needed?) I would need to be given rights to PyPI as well.

**Comment by vidartf** [comment]
Fixed by 733a6c883febaf43d7a905ac0c1208b1aff52c86
